src/create_disk.sh

```diff
@@ -29,6 +29,10 @@ extrakargs="$1" && shift
 
 set -x
 
+# Load policy, so we can do labeling.
```

Comment:
Should this be in the supermin init script in general? Let's not block this PR on it, but there might be other similar bugs lurking. OTOH doing so might also break things.

Reply:
It already is actually; dropped this hunk.